### ISSUE TITLE
Unable to install cypress - Error: Corrupted download

### ISSUE THREAD

**Issue by Nbhatia90** [cypress-io/cypress#26228]
### Current behavior

I am trying to install cypress using following command in the terminal (I already have node js installed)
npm install cypress --save-dev
But Installation gets stuck and ends up with error saying Corrupted download.
URL: https://download.cypress.io/desktop/12.8.1?platform=win32&arch=x64
If I try to do a direct download from above URL then also it fails with Network Error . 
[2023-03-27T08_17_36_184Z-debug-0.log](https://github.com/cypress-io/cypress/files/11076458/2023-03-27T08_17_36_184Z-debug-0.log)

### Debug logs

```Text
npm ERR! [STARTED] Task without title.
npm ERR! [TITLE]  Downloading Cypress      0% 0s
npm ERR! [TITLE]  Downloading Cypress      1% 20s
npm ERR! [TITLE]  Downloading Cypress      1% 30s
npm ERR! [TITLE]  Downloading Cypress      1% 40s
npm ERR! [TITLE]  Downloading Cypress      1% 50s
npm ERR! [TITLE]  Downloading Cypress      2% 30s
npm ERR! [TITLE]  Downloading Cypress      2% 35s
npm ERR! [TITLE]  Downloading Cypress      2% 40s
npm ERR! [TITLE]  Downloading Cypress      3% 29s
npm ERR! [TITLE]  Downloading Cypress      3% 33s
npm ERR! The Cypress App could not be downloaded.
npm ERR!
npm ERR! Does your workplace require a proxy to be used to access the Internet? If so, you must configure the HTTP_PROXY environment variable before downloading Cypress. Read more: https://on.cypress.io/proxy-configuration
npm ERR!
npm ERR! Otherwise, please check network connectivity and try again:
npm ERR!
npm ERR! ----------
npm ERR!
npm ERR! URL: https://download.cypress.io/desktop/12.8.1?platform=win32&arch=x64
npm ERR! Error: Corrupted download
npm ERR!
npm ERR! Expected downloaded file to have checksum: 64eca5567f50e036b6ee11e4fbe4c0f7f3f4ee097bb7f1264fbef9106e91833a8c8f1c06855189ef9d0efca7066c54ac6fe9c7a8e57f59fa5394738d943a6dca
npm ERR! Computed checksum: 2b813e912e77907282770158d5aa513df49dacd69a13c621672c0740ae99888e47a47e1aca0cbdf72b2941c3137d711d6c520d4d630da1529d31107c80f454a8
npm ERR!
npm ERR! Expected downloaded file to have size: 179318639
npm ERR! Computed size: 5765410
npm ERR!
npm ERR! ----------
npm ERR!
npm ERR! Platform: win32-x64 (10.0.22621)
```

### Cypress Version

12.8.1 

### Node version

v18.15.0

### Package Manager

npm

### Package Manager Version

9.5.0

### Operating system

Linux

### Operating System Version

Windows 11 Os build 22621.1413

**Comment by MikeMcC399** [comment]
@Nbhatia90 

I tried the URL https://download.cypress.io/desktop/12.8.1?platform=win32&arch=x64 which you quoted, and I had no problem to download Cypress via IPv4 and IPv6.

Is the issue still happening? How are you connected to the Internet? Can you try out a different connection or a different computer?

If you execute 

```shell
tracert download.cypress.io
```

at a command prompt of your Windows 11 system, are you getting connected to the server?

**Comment by MikeMcC399** [comment]
I noticed in the logs that you are using OneDrive. You might try downloading Cypress into a folder which is not sync'ed through OneDrive in case you have some network issues due to load.

**Comment by Nbhatia90** [comment]
Hello @MikeMcC399 
Thank you for your suggestions. Issue is resolved now . :)
I tried following steps :
Moved my Project out of One Drive 
I deleted the node_modules folder.
Checked tracert download.cypress.io -- All looked good here
Connected to different network - basically my hotspot from my mobile network .
---This again gave me the same error , plus network was too slow. 
Again switched to home wifi 
Created a new project altogether started from npm init -y , followed by npm install cypress --- But got same issue again with corrupted download.
Switched back to previous project and tried npm install cypress again.  ----This time it downloaded and installed cypress .
Created new project to reproduce the issue and tried to install cypress for it  --- it worked fine. 

Unsure if it was One drive that was causing the download to fail or not. 
Might try again later with enabling One drive to see if I can recreate it.
